pr_agent/algo/utils.py

```diff
@@ -375,7 +375,7 @@ def get_user_labels(current_labels: List[str] = None):
 
 def get_max_tokens(model):
     settings = get_settings()
-    max_tokens_model = MAX_TOKENS[model]
+    max_tokens_model = MAX_TOKENS.get(model, 4000)
```

Comment:
**Suggestion:** Consider adding a default value for `MAX_TOKENS` in the `get_max_tokens` function. This will make the function more robust in case the `MAX_TOKENS` dictionary doesn't contain the model as a key. [enhancement]
```suggestion
    DEFAULT_MAX_TOKENS = 4000
    max_tokens_model = MAX_TOKENS.get(model, DEFAULT_MAX_TOKENS)
```